app/helpers/news_helper.rb

```diff
@@ -8,10 +8,18 @@ def news_article_class
   end
 
   def news_articles(section: nil)
-    if section.present?
-      section.news_articles.published.by_release_date.by_published_at
-    else
-      NewsArticle.published.by_release_date.by_published_at
-    end
+    query_root = if section.present?
+                   section.news_articles
+                 else
+                   NewsArticle
+                 end
+    query_root
+      .preload([
```

Comment:
I only just came across it. AFAIK it does exactly the same as `includes` but I think the name is more intention-revealing. From the docs: